source/_integrations/gryfsmart.markdown

```
@@ -0,0 +1,158 @@
+ha_codeowners:
+    - "@karlowiczpl"
+ha_integration_type: hub
+---
+
+## The [GryfSmart](https://gryfsmart.pl) integration for home assistant allows you to connect to GryfSmart devices
+
+This document describes how to configure and use the GryfSmart integration. The integration support Config Flow (UI-based setup), and requires RS232 interface.
+
+There is currently support for the following device types within Home Assistant:
+
+ - Light
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)